_includes/eiptable.html

```diff
@@ -11,7 +11,7 @@
   {% assign eips = include.eips|where:"status",status|sort:"eip" %}
   {% assign count = eips|size %}
   {% if count > 0 %}
-    <h2>{{status}}</h2>
+    <h2 id="{{status|slugify}}">{{status}}</h2>
```

Comment:
Why?

Reply:
Nice, can we merge this line at least? :)

Reply:
Moved to #2277.